client/blocks/plan-storage/tooltip.jsx

```diff
@@ -0,0 +1,27 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import {
+	useTooltipState,
+	Tooltip as ReakitTooltip,
+	TooltipReference,
+	TooltipArrow,
+} from 'reakit/Tooltip';
+
+function Tooltip( { children, title, ...props } ) {
+	const tooltip = useTooltipState( { placement: 'top-end', gutter: 14 } );
```

Comment:
I realize Reakit's docs define this constant as `tooltip`, but how would you feel about being a little more descriptive here to differentiate it from all the other Tooltip* mentions? `tooltipState` or something?

Reply:
Yep! That's how we name it on Gutenberg too: https://github.com/WordPress/gutenberg/blob/132a01c63e79bfe4514d857f59ed6c39204381f2/packages/components/src/toolbar/toolbar-container.js#L21-L25